blocks/core/ff_module/ff_module-task-event/src/TaskEventContainer.rt

```diff
@@ -0,0 +1,6 @@
+<div
+    class="ff_module-task-event ff_module-task-event--{this.props.event.type}"
+    >
+    <time class="ff_module-task-event__sent">{this.props.event.sent}</time>
```

Comment:
I don't know of anywhere in Firefly that already does it, but it's probably worth checking. Having it in the pattern library seems fine to me.